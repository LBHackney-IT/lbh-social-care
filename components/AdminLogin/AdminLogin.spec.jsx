import { render } from '@testing-library/react';
import AdminLogin from './AdminLogin';

describe('AdminLogin component', () => {
  const props = {
    submitText: 'Foo',
    gssoUrl: 'http//foo/bar',
  };
  it('should render properly', () => {
    const { asFragment } = render(<AdminLogin {...props} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
