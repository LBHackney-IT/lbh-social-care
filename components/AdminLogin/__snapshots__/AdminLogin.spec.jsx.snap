// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminLogin component should render properly 1`] = `
<DocumentFragment>
  <div>
    <a
      class="govuk-button"
      href="http//foo/bar"
    >
      Foo
    </a>
  </div>
</DocumentFragment>
`;
